velox/row/tests/UnsafeRowSerdeTest.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <optional>
+
+#include "velox/row/UnsafeRowDeserializers.h"
+
+#include "velox/row/UnsafeRowFast.h"
+#include "velox/vector/tests/utils/VectorTestBase.h"
+
+using namespace facebook::velox;
+
+namespace facebook::velox::row {
+class UnsafeRowSerdeTests : public testing::Test, public test::VectorTestBase {
+ public:
+  UnsafeRowSerdeTests() {}
+
+  RowVectorPtr createInputRowWithUnknownType(
+      int32_t batchSize,
+      std::function<bool(vector_size_t /*row*/)> isNullAt = nullptr) {
```

Comment:
isNullAt is never specified and it seems not needed; top-level RowVector should not have nulls